profiler_measurements/highDroppingRateScenario.h

```diff
@@ -0,0 +1,71 @@
+#ifndef PIPERT_HIGHDROPPING_SCENARIO_H
+#define PIPERT_HIGHDROPPING_SCENARIO_H
+#include <fcntl.h>
+#include <termio.h>
+#include <unistd.h>
+
+#include <algorithm>
+#include <iostream>
+#include <string>
+#include <thread>
+#include <math.h>
+
+#include "pipert/Profiler.h"
+#include "pipert/Scheduler.h"
+#include "utils.h"
+
+/// Measurement Title : drop rate
```

Comment:
Same as in other file. We don't need the namepace, you should doccomment the classes, not the namespace, you should put the implementation into cpp files and you should check google c++ coding convetions because it is breached here several times in naming.